tests/requirements/dev.txt

```diff
@@ -1,3 +1,4 @@
-django-celery==3.0.23
-djtables==0.1.2
-mock==1.0.1
+django-celery>=3.0.23
```

Comment:
Good idea. We actually don't require celery or django-celery in order to install rapidsms, just for the tests. I'll make it more explicit.